internal/watcher/health/nginx_health_watcher_operator.go

```diff
@@ -7,9 +7,12 @@ package health
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"strings"
 
+	process2 "github.com/shirou/gopsutil/v3/process"
+
 	mpi "github.com/nginx/agent/v3/api/grpc/mpi/v1"
 	"github.com/nginx/agent/v3/internal/datasource/host/exec"
 	"github.com/nginx/agent/v3/internal/watcher/process"
```

Comment:
personally, would prefer to rename this import to process_watcher over renaming the gopsutil one

Reply:
This is also a clear downside to having each watcher in its own package instead of just having a flat `watcher` package. Was there some specific other reason for splitting each into its own directory, other than for cleaner organization purposes?